pkg/util/utils.go

```diff
@@ -723,3 +726,145 @@ func SocketPath() (string, error) {
 	// Glue the socket path together
 	return filepath.Join(xdg, "podman", "podman.sock"), nil
 }
+
+// ImageScpOptions provide options for securely copying images to and from a remote host
+type ImageScpOptions struct {
+	// Remote determines if this entity is operating on a remote machine
+	Remote bool `json:"remote,omitempty"`
+	// File is the input/output file for the save and load Operation
+	File string `json:"file,omitempty"`
+	// Quiet Determines if the save and load operation will be done quietly
+	Quiet bool `json:"quiet,omitempty"`
+	// Image is the image the user is providing to save and load
+	Image string `json:"image,omitempty"`
+	// Transfer determines whether we are transferring an image locally
+	Transfer bool `json:"transfer,omitempty"`
+	// User is used in conjunction with Transfer to determine if a valid user was given to save from/load into
+	User string `json:"user,omitempty"`
+}
+
+// ParseImageSCPArgs returns the valid connection, source, and destination data based off of the information provided by the user
+// args is an array of the command arguments returned are two filled out source and destination options structs as well as a connections array and an error if applicable
+func ParseImageSCPArgs(args []string) (*ImageScpOptions, *ImageScpOptions, []string, error) {
+	source := ImageScpOptions{}
+	dest := ImageScpOptions{}
+
+	cliConnections := []string{}
+	switch len(args) {
+	case 1:
```

Comment:
I thought we agreed, that if there was only one arg it is a error?

Reply:
the syntax `root@localhost::alpine` should work and assume the user to be the default rootless user i think. this syntax can also be used like `podman image scp foobar@192.168.68.122::alpine` meaning we want to load an image from that host